soc/arm/st_stm32/stm32h7/Kconfig.soc

```diff
@@ -78,3 +78,8 @@ config SOC_STM32H7B3XXQ
 	select CPU_HAS_FPU_DOUBLE_PRECISION
 
 endchoice
+
+config STM32H7_DUAL_CORE
+	bool "Dual Core"
+	depends on SOC_SERIES_STM32H7X
+	default y
```

Comment:
Don't default to `y`.
Use `select STM32H7_DUAL_CORE` in matching SoC definitins, like:
```
config SOC_STM32H747XX
	bool "STM32H747XX"
	select CPU_HAS_FPU_DOUBLE_PRECISION if CPU_CORTEX_M7
        select STM32H7_DUAL_CORE
```

(might require to move `STM32H7_DUAL_CORE`  definition before the choice section.

Reply:
ok